tests/fixtures/hello_world/hello_world.hocon

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+# This file defines everything necessary for a data-driven test.
+{
+    # Describes what agent to test against.
+    "agent": "hello_world",
+
+    # Needs to success N times out of M
+    "success_ratio": "3/3", 
+
+    # Interactions are a series of dictionaries with request elements paired with
+    # descriptions of response checks.
+    "interactions": [
+        {
+            # This is what we send as input to streaming_chat()
+            "text": """
+From earth, I approach a new planet and wish to send a short 2-word greeting to the new orb.
+""",
```

Comment:
A test case to help make hello_world better.
Note that hello_world, while it has improved input and prompting is still not 100% yet,
so this is not part of the unit tests at all.